### PR TITLE
Rearrange the docs

### PR DESCRIPTION
**Pull request by IwonaLanger**
**Description**

Changes proposed in this pull request:

- Add the `user` subfolder to the `docs` directory
- Put the docs in the `contributor` or `user` subfolders
- Update terminology
- Add the table with experimental plans
- Fix links
- Fix formatting
- Edit the EU access document (move part of the doc to internal documentation)

**Related issue(s)**
#26

### CODE REVIEW

**Review by PK85 — DISMISSED** [review]

File: docs/user/05-10-provisioning-kyma-environment.md [hunk 1]
```diff
@@ -64,11 +64,11 @@ This tutorial shows how to provision SAP BTP, Kyma runtime on Azure using Kyma E
...
    ```  
 
-3. Check the operation status as described [here](08-03-operation-status.md).
+4. Check the operation status as described in the [Check operation status](05-30-operation-status.md) document.
 
-## BTP Operator
+## SAP BTP Service Operator
 
-If you need a BTP Operator component installed, obtain [BTP Operator access credentials](https://github.com/SAP/sap-btp-service-operator/blob/v0.2.5/README.md#setup) and provide them in the provisioning request. See the following example:
+If you need a SAP BTP Service Operator component installed, obtain [SAP BTP Service Operator access credentials](https://github.com/SAP/sap-btp-service-operator/blob/v0.2.5/README.md#setup) and provide them in the provisioning request. See the following example:
```
Comment (nataliasitko) [inline docs/user/05-10-provisioning-kyma-environment.md:71]:
```suggestion
If you need the SAP BTP Service Operator component installed, obtain [SAP BTP Service Operator access credentials](https://github.com/SAP/sap-btp-service-operator/blob/v0.2.5/README.md#setup) and provide them in the provisioning request. See the following example:
```

File: docs/user/05-40-instance-details.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ This tutorial shows how to get the SAP BTP, Kyma runtime instance details.
...
+1. Export the instance ID that you set during [provisioning](05-10-provisioning-kyma-environment.md):
 
    ```bash
    export INSTANCE_ID={SET_INSTANCE_ID}
    ```
 
    > **NOTE:** Ensure that the BROKER_URL and INSTANCE_ID environment variables are exported as well before you proceed.
 
-2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](03-05-authorization.md) to verify that provisioning/deprovisioning succeeded:
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](../contributor/01-10-authorization.md) to verify that provisioning/deprovisioning succeeded:
```
Comment (nataliasitko) [inline docs/user/05-40-instance-details.md:15]:
```suggestion
2. Make a call to Kyma Environment Broker with a proper **Authorization** [request header](../contributor/01-10-authorization.md) to verify that provisioning/deprovisioning succeeded:
```

File: docs/user/05-50-configure-list-of-modules.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Configure list of modules
+
+By default, Kyma Environment Broker applies the Kyma CR, including the default modules selected by Kyma, to a cluster.
```
Comment (nataliasitko) [inline docs/user/05-50-configure-list-of-modules.md:3]:
```suggestion
By default, Kyma Environment Broker applies the Kyma custom resource, including the default modules selected by Kyma, to a cluster.
```

File: docs/user/05-50-configure-list-of-modules.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+You can configure your custom list of modules by setting the "modules" object in the JSON schema.
```
Comment (nataliasitko) [inline docs/user/05-50-configure-list-of-modules.md:4]:
```suggestion
You can configure your custom list of modules by setting the **modules** object in the JSON schema.
```
I'm not sure about this, but I guess we could use bold font for "modules" and "oneof". I think they're both JSON Schema keywords (for example, see this [doc](https://json-schema.org/blog/posts/applicability-json-schema-fundamentals-part-1#putting-everything-together-avoiding-oneof-pitfalls))

File: docs/user/05-50-configure-list-of-modules.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+To do that, use the following two fields:
+- **default** (bool) - defines whether to use the default list of modules
+- **list** (array) - defines a custom list of modules
+
+API for module configuration is built on the "oneOf" feature from the JSON schema. If the "modules" object is passed to API, it must have only one valid option. Thus, to pass JSON API Validator, you must set only one field. See examples below.
```
Comment (nataliasitko) [inline docs/user/05-50-configure-list-of-modules.md:9]:
```suggestion
API for module configuration is built on the **oneOf** feature from the JSON schema. If the **modules** object is passed to API, it must have only one valid option. Thus, to pass JSON API Validator, you must set only one field. See examples below.
```

File: docs/user/05-50-configure-list-of-modules.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+   ```
+   "modules": {
+       "default": false
+   }
+   ```
+
+## Incorrect API calls
+
+- A call with the **modules** section empty fails due to using the "oneOf" feature in the JSON schema.
```
Comment (nataliasitko) [inline docs/user/05-50-configure-list-of-modules.md:54]:
```suggestion
- A call with the **modules** section empty fails due to using the **oneOf** feature in the JSON schema.
```

File: docs/user/05-50-configure-list-of-modules.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+   ```
+   "modules": {}
+   ```
+
+- A call with the **modules** section with both parameters filled in fails due to using the "oneOf" feature in the JSON schema.
```
Comment (nataliasitko) [inline docs/user/05-50-configure-list-of-modules.md:60]:
```suggestion
- A call with the **modules** section with both parameters filled in fails due to using the **oneOf** feature in the JSON schema.
```

**Review by nataliasitko — APPROVED** [review]

File: docs/user/05-50-configure-list-of-modules.md (hunk 4 above)
Comment (nataliasitko) [reply docs/user/05-50-configure-list-of-modules.md]: Thank you for clarifying that

**Pull request merged**